WcaOnRails/lib/database_dumper.rb

```diff
@@ -1,10 +1,21 @@
 # frozen_string_literal: true
 
 module DatabaseDumper
-  JOIN_WHERE_VISIBLE_COMP = "JOIN Competitions ON Competitions.id=competition_id WHERE showAtAll=1"
-  DUMP_TIMESTAMP_NAME = "developer_dump_exported_at"
-  VISIBLE_ACTIVITY_IDS = "SELECT A.id FROM schedule_activities AS A JOIN venue_rooms ON (venue_rooms.id = holder_id AND holder_type='VenueRoom') JOIN competition_venues ON competition_venues.id = competition_venue_id #{JOIN_WHERE_VISIBLE_COMP}"
-                         .freeze
+  WHERE_VISIBLE_COMP = "WHERE Competitions.showAtAll=1"
```

Comment:
why are we freezing only some of the strings here? They are all constants right?

Reply:
Because constants that only contain string values, like
```ruby
  MY_CONST = "some_value"
```
are automatically frozen. Only the constants that have interpolation, like
```ruby
  MY_CONST = "some_value"
  MY_OTHER_CONST = "#{MY_CONST} yet again"
```
need manual freezing.

Reply:
Cool, Thanks!